src/logic/safe/store/reducer/gatewayTransactions.ts

```diff
@@ -51,28 +49,27 @@ export const gatewayTransactionsReducer = handleActions<GatewayTransactionsState
       const newHistory: StoreStructure['history'] = cloneDeep(state[chainId]?.[safeAddress]?.history || {})
 
       values.forEach((value) => {
-        if (isDateLabel(value)) {
+        if (!isTransactionSummary(value)) {
```

Comment:
Why change this? Seems unrelated?

Reply:
It was unnecessarily checking if not a date label and that it is a summary. Now it just checks that it is a summary?